UnitySDK/Assets/ML-Agents/Scripts/Grpc/RpcCommunicator.cs

```diff
@@ -136,6 +140,7 @@ void UpdateEnvironmentWithInput(UnityRLInputProto rlInput)
         {
             SendRLInputReceivedEvent(rlInput.IsTraining);
             SendCommandEvent(rlInput.Command, rlInput.EnvironmentParameters);
+            SendSideChannelData(m_SideChannelsDict, rlInput.SideChannel.ToArray());
```

Comment:
Should we rename these methods to "Process*" or "Handle*" or something like that? "Send" is misleading since it sounds like (to me at least) it's sending it to the other process.